src/styles/global/index.scss

```diff
@@ -1,4 +1,5 @@
 @import 'variables';
 @import 'media';
 @import 'mixins';
-@import 'flex';
\ No newline at end of file
+@import 'flex';
+@import 'datepicker';
```

Comment:
We shouldn't put this one here 😄
This way we will duplicate the datepicker styles (the `.dp__input` class) for *every* component we display (sounds like a peculiar bug 😱).
This file (as I can see you append an import to it to all your components) is for those reusable **sass** variables or mixins (and sass is the key here - as this does not generate some css under the hood as an effect)
Put in your `App.vue` or `main.js` instead.

Reply:
This sounds quite bad :fearful: - did not think about it this way and went for a sass partial only in order to keep the styles flow in check.
But I definitely don't want to duplicate so many times.. will change the import destination.

Reply:
That actually sometimes happens - hence we should be careful with such tricks to globally import things in our components 😄
Wouldn't be a big hurt and probably you wouldn't notice it until you would look in the inspector and see duplicated styles but still - we have to be careful 😄